examples/cpp/main.cpp

```diff
@@ -36,12 +36,12 @@ int main(int argc, char *argv[])
     QMenuBar *menuBar = new QMenuBar(mainWindow);
     QMenu *actionsMenu = new QMenu(QStringLiteral("Actions"), menuBar);
     menuBar->addMenu(actionsMenu);
-    actionsMenu->addAction(QStringLiteral("Find..."), console, &QTermWidget::toggleShowSearchBar,
-                           QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_F));
-    actionsMenu->addAction(QStringLiteral("Copy"), console, &QTermWidget::copyClipboard,
-                           QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_C));
-    actionsMenu->addAction(QStringLiteral("Paste"), console, &QTermWidget::pasteClipboard,
-                           QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_V));
+    actionsMenu->addAction(QStringLiteral("Find..."), QKeySequence(QLatin1String("Ctrl+Shift+F")), 
+        console, &QTermWidget::toggleShowSearchBar);
+    actionsMenu->addAction(QStringLiteral("Copy"), QKeySequence(QLatin1String("Ctrl+Shift+C")), 
+        console, &QTermWidget::copyClipboard);
+    actionsMenu->addAction(QStringLiteral("Paste"), QKeySequence(QLatin1String("Ctrl+Shift+V")), 
+        console, &QTermWidget::pasteClipboard);
```

Comment:
@yan12125 `Qt::CTRL | Qt::SHIFT | Qt::Key_V` does indeed work. Even, `QKeySequence(QLatin1String("Ctrl+Shift+C"))` is a valid way to define keybinds, although it was preferable to use `QKeySequence(tr("Ctrl+Shift+C", "Edit|Copy"))` so that it could get translated suitably.

Reply:
I am OK with either updating this or leaving it alone.  It worked great for me but I haven't tested it with German locale.

Reply:
Oh, I didn't notice this is just an example. Let's use constants like `Qt::CTRL` as it was (with `|` operator, as using `+` operator is deprecated).

If someone wants working shortcuts in the example with different locales, it can be changed later.

Reply:
OK.  I think I can do that.

Reply:
Should be as you want it now.